includes/definitions/adva-fsp150cp.yaml

```diff
@@ -0,0 +1,19 @@
+os: adva-fsp150cp
+text: 'ADVA FSP150CP'
+type: network
+icon: adva
+group: ADVA
+ifname: true
+mib_dir: adva
+over:
+    - { graph: device_bits, text: 'Device Traffic' }
+discovery:
+    -
+        sysObjectID:
+            - .1.3.6.1.4.1.2544.1.10.1.1
+discovery_modules:
+    cisco-vrf-lite: false
+    storage: false
+    hr-device: false
+    arp-table: false
+    ucd-diskio: false
```

Comment:
Any specific reason to disable discovery modules

Reply:
Those devices are very old (20 years, EOL, EOS) and are very sensible to crashes when stressed. As they are pure L1 transport (basically a media converter with a GUI), none of these modules will ever give a result. And no update will ever come to extend :D